contents/naturalLanguage/nl.md

```diff
@@ -0,0 +1,195 @@
+<frontmatter>
+  title: Introduction to Natural Language Processing (NLP)
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Natural Language Processing (NLP)
+Authors: [Lum Ka Fai Jeffry](https://github.com/j-lum)
+
+* [What is NLP?](#what-is-nlp)
+* [Core problems in NLP](#core-problems-in-nlp)
+   * [Syntax](#syntax)
+   * [Semantics](#semantics)
+   * [Relations](#relations)
+* [Applications of NLP](#applications-of-nlp)
+* [What's next](#whats-next)
+* [References](#references)
+
+
+## What is NLP?
+Natural Language Processing (NLP) is the set of methods for making human language accessible to computers.
```

Comment:
```suggestion
Natural Language Processing (NLP) is the set of methods that makes human language accessible to computers.
```

Reply:
"for making" has the implication that the methods were tailored for the said purpose.
"that makes" loses a bit of that meaning IMO.

Reply:
Ah I got your point. I just feel the sentence is a bit weirdly framed.